addons/point_of_sale/static/src/app/screens/ticket_screen/ticket_screen.js

```diff
@@ -333,7 +333,8 @@ export class TicketScreen extends Component {
         return this._getOrderList().filter(predicate);
     }
     getDate(order) {
-        return deserializeDate(order.validation_date).toFormat("yyyy-MM-dd HH:mm a");
+        // keeping the same logic as backend
+        return deserializeDate(new Date(order.creation_date).toISOString().replace('T', ' ').slice(0,19)).toFormat("yyyy-MM-dd HH:mm a");
```

Comment:
Here, since validation_date is already deserialized (already a luxon date object), then you can directly call toFormat.

Reply:
Hello @caburj 

We cannot use the validation date since the order validation has not been completed yet. Therefore, we are using the creation date instead of the validation date. If we were to use the validation date, an issue would arise during order creation, as the validation date is not defined at that point. However, after refreshing the page, the validation date becomes available from the init_from_JSON method.

However, the creation date format, such as 'Mon Aug 21 2023 12:46:57 GMT+0530 (India Standard Time),' isn't compatible with the deserializeDate() function's '2018-01-01 00:00:00' format. This issue is specific to this part of the code. Any suggestions for a better solution would be appreciated."

Thanks!

Reply:
We can keep the current behavior in 16.0 where the current datetime is used by "moment" lib when validation_date is undefined. Basically, the current datetime is the fallback. It will be something like:

```suggestion
        const date = order.validation_date ? deserializeDate(date) : luxon.DateTime.now();
        return date.toFormat("yyyy-MM-dd HH:mm a");
```

In fact, we need to establish a convention. For all model instances, date props should be luxon.DateTime. And when we need to put it to localStorage or send it to the network (by making rpc), we serialize it using the one from "@web/core/l10n/dates" module. And if we want to display it, we call .toFormat on the date object.

Reply:
Hello @caburj 
Using luxon.DateTime.now() would consistently provide the current datetime, without refreshing the page, which is not the desired behavior. This issue can be observed in the demo video: https://www.awesomescreenshot.com/video/20097045?key=a476637d0172c70257bc95d5aba2c752

Reply:
Hello @caburj

Reply:
The idea is to restore the behavior in 16.0 such that if "validation_date" is undefined, we show the current datetime.

Reply:
Are you on discord? Can we talk in real-time for this task?